lib/source-destination/block-device.ts

```diff
@@ -18,7 +18,6 @@ import { delay, promisify } from 'bluebird';
 import { Drive as DrivelistDrive } from 'drivelist';
 import { ReadResult, WriteResult } from 'file-disk';
 import { unmountDisk } from 'mountutils';
-import { platform } from 'os';
```

Comment:
I'd prefer using `os.platform()` as we can easily mock it in the tests.

Reply:
Yup, added it back